### DISCUSSION TITLE
Farming address changed with no input from me: "No private keys for one or both addresses"

### DISCUSSION THREAD

**Discussion by NightFlightELR** [Chia-Network/chia-blockchain#4609] [Support]
Hi I am very enthusiastic about Chia!

Please help, I has been a long time since I received reward. Currently only 2 Chia farmed [1st image]. I have 3 PCs with same address (entered same 24 words mnemonic) and the Private key with public fingerprint is the same.

On one PC everything is OK if I check Manage Farming Reward.
But on other one it is not (I have not changed it, no one has access to my PC but me, upgraded only official versions): _*I have triangle and "No private keys for one or both addresses. Safe only..." [2nd image]*_

When I check that address in Chia Explorer it has 4 Chia [3rd image]. How do I access them? I have not changed the farming address at any time.

Are the 4 coins lost :(

![image](https://user-images.githubusercontent.com/84044502/117881060-17983400-b2a9-11eb-814f-894074bb8a42.png)

![image](https://user-images.githubusercontent.com/84044502/117881021-094a1800-b2a9-11eb-8b44-434319ecdaf5.png)

![image](https://user-images.githubusercontent.com/84044502/117881196-3eef0100-b2a9-11eb-8567-1ba1c0dc7f6c.png)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
